def.bzl

```diff
@@ -30,20 +30,22 @@ project = struct(
     ),
     kubernetes = struct(
         version = "1.14.6",
-        platforms = [
-            {
-                "platform": "linux",
-                "sha256": "5f8e8d8de929f64b8f779d0428854285e1a1c53a02cc2ad6b1ce5d32eefad25c",
-            },
-            {
-                "platform": "darwin",
-                "sha256": "de42dd22f67c135b749c75f389c70084c3fe840e3d89a03804edd255ac6ee829",
-            },
-            {
-                "platform": "windows",
-                "sha256": "3aa2d64f5eb9564622ddabe5f0a6c12d13d9dda90125f5a56ce41779395fa6f5",
-            },
-        ],
+        kubectl = struct(
+            platforms = struct(
+                darwin = {
+                    "url": "https://storage.googleapis.com/kubernetes-release/release/v1.14.6/bin/linux/amd64/kubectl",
```

Comment:
This no longer enforces the same version binary across platforms. Up to you to decide if that's better or worse; I guess it could be either…

Reply:
I think this is ok. IMO, it's pretty obvious that we should bump all platforms at once.